Echo/ApplicationInputHandler.cpp

```diff
@@ -0,0 +1,183 @@
+#include "PolySyncDataModel.hpp"
+#include "ApplicationInputHandler.hpp"
+#include "getopt.h"
+#include <algorithm>
```

Comment:
Small style thing. Should library <includes> be before the polysync / example includes?